public/app/features/dashboard/state/PanelModel.ts

```diff
@@ -672,6 +672,12 @@ export class PanelModel implements DataConfigSource, IPanelModel {
     }
     this.libraryPanel = libPanel;
   }
+
+  unlinkLibraryPanel() {
+    delete this.libraryPanel;
+    this.configRev++;
```

Comment:
So not increasing the `configRev` means that the change is not getting committed after applying it on the panel?

Reply:
yes, related to this condition here: https://github.com/grafana/grafana/blob/main/public/app/features/dashboard/components/PanelEditor/state/actions.ts#L119

Reply:
Thank you!